albumentations/augmentations/transforms.py

```diff
@@ -1554,6 +1554,40 @@ def get_transform_init_args_names(self):
         return ('hue_shift_limit', 'sat_shift_limit', 'val_shift_limit')
 
 
+class Solarize(ImageOnlyTransform):
+    """Invert all pixel values above a threshold.
+
+        Args:
+            threshold ((int, int) or int, or (float, float) or float): range for solarizing threshold.
+            If threshold is a single value, the range will be [threshold, threshold]. Default: 128.
+            p (float): probability of applying the transform. Default: 0.5.
+
+        Targets:
+        image
+
+        Image types:
+            any
+
+        """
+    def __init__(self, threshold=128, always_apply=False, p=0.5):
+        super(Solarize, self).__init__(always_apply, p)
+
+        if isinstance(threshold, (int, float)):
+            threshold = max(0, threshold)
+            self.threshold = (threshold, threshold)
+        else:
+            self.threshold = to_tuple(threshold, low=0)
```

Comment:
can we use just `to_tuple(threshold, low=threshold)` here?

Reply:
Yes it is look better